src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -44,6 +47,10 @@
     public int minIslandName = 3;
     public int teleportDelay = 0;
 
+    public Map<Integer, String> islandRankDisplayNames = Arrays.stream(IslandRank.values()).collect(
+            Collectors.toMap(IslandRank::getLevel, rank ->
+                    WordUtils.capitalizeFully(rank.name().replace('_', ' ')).replace(' ', '-')));
```

Comment:
Stream API ❤️ 

Anyways, this should probably be hardcoded. Sure, it looks nice and is less work but we hardcode all configs.

Reply:
Why do we hardcode all configs?

Reply:
Most of the time it's just the only option. It's also easier to read and understand. We just always did it, I don't see the point of not hardcoding this option.

Reply:
I removed the soft coding from Inventories.java. But my heart isn't in the right place if I remove that in the Configuration.java

Reply:
Ye im fine with it being like this tbh, but people are stupid? what if they accidently delete something from this? We need a way to check and update if the islandRankDisplayNames doesnt contain a key

Reply:
I agree, we should have some sort of fallback so that there are always 5 rank names. If we use this code, please reformat it... collect() on a new line and do something about that Collectors.toMap() parameter list

Reply:
Sorry took me a while but I changed it.